spark/src/main/scala/geotrellis/spark/io/hadoop/HadoopSparkContextMethods.scala

```diff
@@ -56,24 +70,13 @@ trait HadoopSparkContextMethods {
   def hadoopMultiBandGeoTiffRDD(path: String, tiffExtensions: Seq[String]): RDD[(ProjectedExtent, MultiBandTile)] =
     hadoopMultiBandGeoTiffRDD(new Path(path), tiffExtensions)
 
-  def hadoopMultiBandGeoTiffRDD(path: Path, tiffExtensions: Seq[String] = defaultTiffExtensions): RDD[(ProjectedExtent, MultiBandTile)] = {
-    val searchPath = path.toString match {
-      case p if tiffExtensions.exists(p.endsWith) => path
-      case p =>
-        val extensions = tiffExtensions.mkString("{", ",", "}")
-        new Path(s"$p/*$extensions")
-    }
-
-    val updatedConf =
-      sc.hadoopConfiguration.withInputDirectory(searchPath)
-
+  def hadoopMultiBandGeoTiffRDD(path: Path, tiffExtensions: Seq[String] = defaultTiffExtensions): RDD[(ProjectedExtent, MultiBandTile)] =
```

Comment:
It may be sensible to remove spaces around the `=` on a default param - with word wrap, this can look pretty confusing.